staging/src/k8s.io/apiserver/pkg/util/flowcontrol/metrics/metrics.go

```diff
@@ -227,7 +227,7 @@ var (
 			Subsystem:      subsystem,
 			Name:           "request_concurrency_limit",
 			Help:           "Shared concurrency limit in the API Priority and Fairness subsystem",
-			StabilityLevel: compbasemetrics.ALPHA,
+			StabilityLevel: compbasemetrics.BETA,
```

Comment:
Since we are deprecating this metric and plan to delete it, I think that we should promote the equal metric `apiserver_flowcontrol_nominal_limit_seats` to BETA instead of this one.

Reply:
Kept this as alpha and updated `nominal_limit_seats` to BETA